test/test_helpers/publishing_api_test.rb

```diff
@@ -328,19 +328,15 @@
   end
 
   describe "#stub_publishing_api_isnt_available" do
-    it "returns a 503 for V2 requests" do
+    it "returns a 503 for any request" do
       stub_publishing_api_isnt_available
 
       assert_raises GdsApi::BaseError do
-        publishing_api.get_content_items({})
+        publishing_api.lookup_content_id(base_path: "")
       end
-    end
-
-    it "returns a 503 for V1 requests" do
-      stub_publishing_api_isnt_available
 
       assert_raises GdsApi::BaseError do
-        publishing_api.lookup_content_id(base_path: "")
+        publishing_api.get_content_items({})
```

Comment:
It's odd that these lines have switched order, but ok.

Reply:
Sure, unintentional. I've swapped them back now.